Automattic/wp-calypso
# Add Jetpack Mobile to header navigation

#### Proposed Changes

* This PR adds a link to Jetpack Mobile page to the header navigation on cloud.jetpack.com

Context: pbNhbs-51O-p2 

#### Testing Instructions



* Checkout this branch locally
* Run `yarn start-jetpack-cloud`
* Visit http://jetpack.cloud.localhost:3000/pricing
* Make sure that the Products submenu looks like the screenshot below, and that it matches the proposed design (in the link above)
* Confirm that the link works, and the animation looks good

 
<img width="1906" alt="image" src="https://user-images.githubusercontent.com/5714212/213843189-828585fb-a713-4377-ae3f-9aaa881f7c8d.png">


#### Pre-merge Checklist
